app/src/main/java/org/linphone/LinphoneService.java

```diff
@@ -396,17 +417,17 @@ public synchronized void onDestroy() {
         Core lc = LinphoneManager.getLcIfManagerNotDestroyedOrNull();
```

Comment:
From here the modification allowed the missed call notification to survive the killing of the service